src/pad.c

```diff
@@ -15,6 +18,8 @@
  * Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
+#include "pad.h"
```

Comment:
Why was `pad.h` moved here?

By convention, local inclusion (`"foo.h"`) should be made **after** standard inclusions (`<foo.h>`)